cocotb/result.py

```diff
@@ -129,3 +129,13 @@ class TestSuccess(TestComplete):
 class SimFailure(TestComplete):
     """Exception showing that simulator exited unsuccessfully."""
     pass
+
+
+if sys.version_info < (3,3):
+    class TimeoutError(OSError):
+        pass
```

Comment:
This belongs in https://github.com/cocotb/cocotb/blob/master/cocotb/_py_compat.py, I think